feature/security/gnsi/certz/tests/client_certificates/client_certificates_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_certificates_test
+
+import (
+	context "context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	setupService "github.com/openconfig/featureprofiles/feature/security/gnsi/certz/tests/internal/setup_service"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+var (
+	testProfile = "gNxI"
```

Comment:
Should this profile be created by the testcase and not assumed to be preconfigured?
something like sending an `addProfileReq` before the test executes? Thoughts, @morrowc ?
```
addProfileRequest := &certzpb.AddProfileRequest{SslProfileId: testProfile}
gnsiC.Certz().AddProfile(ctx, addProfileRequest)
```

Reply:
I admit i don't really understand what testProfile is here.

Is the expectation that the client_certificates_test gets run on it's own? or that something reaches in here (and other featureprofile/profile/*/stuff/*_test.go) and runs this thing on it's own? (so either this being hardcoded here is fine, it's just run as a set of 'all the tests', or we should let the caller of the test set this on run)

Reply:
gNxI is the default ssl profile available on the device.

Reply:
We are using the default gnsi profile available on the server , instead of creating a new ssl profile using addProfileRequest. There was no mention on  the README to create a new profile for the testcases.